src/components/central-programs-table.js

```diff
@@ -372,11 +372,20 @@ const CentralProgramsTable = ({ data, labelContent, codes }) => {
       align: "right",
     },
   ]
+
+  const selectRow = {
+    mode: 'checkbox',
+    clickToSelect: true,
+    nonSelectable: [0],
+    nonSelectableClasses: 'row-index-bigger-than-2101'
+  };
+
```

Comment:
was there a problem that popped up that required selecting rows?

Reply:
oh no! I was testing out some other options but I will remove this in a bit.